src/sql-parser/src/parser.rs

```diff
@@ -7053,6 +7053,9 @@ impl<'a> Parser<'a> {
         if self.parse_keyword(TIMESTAMP) {
             self.parse_explain_timestamp()
                 .map_parser_err(StatementKind::ExplainTimestamp)
+        } else if self.parse_keyword(KEY) || self.parse_keyword(VALUE) {
```

Comment:
nit: Instead of using `parse_keyword` here and `prev_token` later in `parse_explain_schema`, you can use `peek_token` here.

Reply:
done!